importer/main.py

```diff
@@ -475,6 +475,77 @@ def care_team_extras(
     return payload_string
 
 
+# This function takes in a payload_obj, a position and a tracker
+# It deletes the object at the given position in the resource->characteristic
+# It then returns the updated payload_obj and a new tracker for the remaining objects
+def delete_missing_obj_from_payload(payload_obj, position, tracker):
+    del payload_obj["resource"]["characteristic"][position]
+    return payload_obj, tracker-1
+
+
+# custom extras for product import
+def product_extras(resource, payload_string):
+    payload_obj = json.loads(payload_string)
+    product_name = resource[0]
+    tracker = 0
+
+    try:
+        (_, active, *_, previous_id, is_attractive_item, availability, condition, appropriate_usage,
+         accountability_period, image_source_url) = resource
+    except ValueError:
+        logging.error("Skipping: " + product_name + " : Because of missing columns ")
+        active = previous_id = is_attractive_item = availability = condition = appropriate_usage = \
+            accountability_period = image_source_url = "missing_column"
```

Comment:
what if you set this to null here? then in the conditional below, you just check that it's not null w/an `if active` etc

Reply:
was trying to differentiate this from an actual null from the csv, but I guess we can just have them be the same